packages/headless/src/controllers/facets/range-facet/date-facet/headless-date-filter.test.ts

```diff
@@ -126,13 +125,22 @@ describe('date filter', () => {
       );
       expect(action).toBeTruthy();
     });
+
+    it('should throw when range is invalid', () => {
+      const value = buildMockDateFacetValue(
+        buildDateRange({start: 1616679091000, end: 1616592691000})
```

Comment:
It's not readily apparent why that's invalid (I'm guessing because end < start). A clearer test name description would help.